### DISCUSSION TITLE
Feature Request - Allow customization of chat bubble per message

### DISCUSSION THREAD

**Discussion by jlubeck** [flyerhq/flutter_chat_ui#92] [Ideas & Requests]
I think I read somewhere that more customization is planned, but just wanted to give my use case in case it requires something special. Basically I'm looking to change the background color of the chat bubbles based on specific flags... For example I want to make the admin bubbles to have a different color than the rest of the bubbles. I assume I would add this flag when creating the types.Message, probably on the Metadata Map? And then there should be some kind of decorator method where we would get the types.Message info so we can modify the decorator...

Hopefully I made sense and you see value in this idea

**Comment by demchenkoalex** [comment]
Released in [v1.4.3](https://github.com/flyerhq/flutter_chat_ui/releases/tag/v1.4.3).
